### ISSUE TITLE
EPIC: Website accessibility

### ISSUE THREAD

**Issue by MrMarlonM** [MrMarlonM/good-world-news#36]
**Short description**

- The website should be clearly structured
- The website should have a clear and concise navigation
- The website should be accessible for screen readers
- The website should be accessible for different screen sizes
